exe/node/Main.hs

```diff
@@ -23,16 +23,16 @@ import qualified Oscoin.P2P as P2P
 import qualified Oscoin.P2P.Handshake as Handshake
 import           Oscoin.Protocol (runProtocol)
 import           Oscoin.Storage (hoistStorage)
-import qualified Oscoin.Storage.Block.STM as BlockStore.Concrete.STM
+import qualified Oscoin.Storage.Block.STM as BlockStore.STM
 import qualified Oscoin.Storage.State as StateStore
 import qualified Oscoin.Telemetry as Telemetry
 import           Oscoin.Telemetry.Logging (withStdLogger)
 import qualified Oscoin.Telemetry.Logging as Log
 import           Oscoin.Telemetry.Metrics
 
 import qualified Control.Concurrent.Async as Async
+import           Control.Monad.Managed
```

Comment:
Apparently there is a `Control.Monad.Managed.Safe` module which is even more stringent -- do we have any chance of using that?

Reply:
It's the same thing, except that the `Safe` module doesn't export `with`, which let's you leak the "managed" resource

Reply:
Gotcha 😉